spec/spec_helper.rb

```diff
@@ -24,7 +24,7 @@
 
 # rubocop:disable Metrics/LineLength
 def with_tokenizing_profile
-  with_env("AU_TNS_PAY_URL", "https://TESTEVHEROSTG02:e750f25acd207d168117d1050e6fbaf7@secure.ap.tnspayments.com") do
+  with_env("AU_TNS_PAY_URL", "https://TESTEVHEROSTG02:e750f25acd207d168117d1050e6fbaf7@ap-gateway.mastercard.com") do
```

Comment:
@HowlingEverett on a related note, I'm not sure it's wise to publish your API credentials on github, even if they are test credentials.